cells/std/lib/default.nix

```diff
@@ -38,4 +38,6 @@ in
       ../lib/ops/mkMicrovm.nix {
         inputs = requireInput {inputs = inputs';} "makes" "github:fluidattacks/makes" "std.std.lib.fromMakesWith";
       };
+
+    mkDevelopDrv = import ../../../src/devshell-drv.nix;
```

Comment:
that is likely not going to work, the source for cells stops at `./cells`.

Reply:
yeah that should be fine